AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithIAMIntegrationTests/README.md

```diff
@@ -0,0 +1,90 @@
+## GraphQL with IAM Auth Integration Tests
+
+The following steps show how to set up an GraphQL endpoint with AppSync. The auth configured will be IAM. This set up is used to run the tests in `GraphQLWithIAMIntegrationTests.swift`.
+
+### Set-up
+
+TODO: Figure out how to use Amplify CLI to  `amplify add api` and select `IAM` auth. Currently it does not set up correct IAM policies. 
+
+
+### Old notes
+
+// MARK: set up
+    /*
```

Comment:
I don't think you meant to have Swift comment doc in the README file? Recommend you also fix indenting, and view the file in a markdown renderer before next commit.

Reply:
will update this to markdown. i had left out this because I added a TODO at the top of this file as it's currently not an easy flow for developers to use the CLI. they have to go into the console and enable iam policy for access to appsync.